python/typedb/common/exception.py

```diff
@@ -40,26 +38,25 @@ class TypeDBDriverExceptionExt(TypeDBDriverException):
         try:
             transaction.commit()
         except TypeDBDriverException as err:
-            println("Error:", err)
+            print("Error:", err)
     """
 
-    def __init__(self, msg: Union[ErrorMessage, str], cause: BaseException = None, params: Any = None):
+    def __init__(self, msg: Union[ErrorMessage, str], params: Any = None):
```

Comment:
We haven't use `cause` anywhere, so we removed it and replaced old usages of `of()` with `__init__()`. New `of()` is used for wrapping native exceptions.